sdk/cli/src/commands/verification-challenge.ts

```diff
@@ -0,0 +1,83 @@
+import { missingApplication, missingPersonalAccessTokenError } from "@/error/missing-config-error";
+import { nothingSelectedError } from "@/error/nothing-selected-error";
+import { blockchainNodePrompt } from "@/prompts/cluster-service/blockchain-node.prompt";
+import { instancePrompt } from "@/prompts/instance.prompt";
+import { serviceSpinner } from "@/spinners/service.spinner";
+import { getInstanceCredentials } from "@/utils/config";
+import { sanitizeAndValidateInstanceUrl } from "@/utils/instance-url-utils";
+import { Command } from "@commander-js/extra-typings";
+import password from "@inquirer/password";
+import { createSettleMintClient } from "@settlemint/sdk-js";
+import { loadEnv } from "@settlemint/sdk-utils/environment";
+import { intro, note, outro } from "@settlemint/sdk-utils/terminal";
+import type { DotEnv } from "@settlemint/sdk-utils/validation";
+
+/**
+ * Creates and returns the 'challenge' command for the SettleMint SDK.
+ * This command outputs the current platform configuration.
+ */
+export function verificationChallengeCommand() {
```

Comment:
it is actually a challenge response we are creating ... (but behind the scenes it also fetches the challenge) ... so maybe verification-response or something would be better? or because it's strict pincode "pincode-response"